SeleniumTestXUnit/Tests/Features/RecycleBin/DeleteAll.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Empty recycle bin
+    As a logged in user, the user should be able to empty the recible bin
+    Background:
+        Given the user is logged in
+        And has deleted items
+
+    @Draft
+    @Smoke
+    Scenario: Empty the recycle bin succesfully
+        When the user clicks the recycle bin context menu
+        And clicks the empty recycle bin button
+        Then the site will empty the recycle bin
```

Comment:
The validation step should follow a team convention format, most likely, always using the word "should", e.g., "Then the recycle bin should not contain any item"